skype/skype-ps/skype/New-CsTeamsIPPhonePolicy.md

```diff
@@ -0,0 +1,223 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+Module Name: Skype for Business Online
+online version: https://docs.microsoft.com/powershell/module/skype/new-csteamsipphonepolicy
+applicable: Skype for Business Online
+title: New-CSTeamsIPPhonePolicy
+author: tonywoodruff
+ms.author: anwoodru
+ms.reviewer: kponnus
+manager: sandrao
+schema: 2.0.0
+---
+
+# New-CsTeamsIPPhonePolicy
+
+## SYNOPSIS
+
+New-CSTeamsIPPhonePolicy allows you to create a policy to manage features related to Teams phone experiences. Teams phone policies determine the features that are available to users.
+
+## SYNTAX
+
+```
+New-CsTeamsIPPhonePolicy [-Description <Object>] [-HotDeskingIdleTimeoutInMinutes <Object>]
+ [-AllowHotDesking <Object>] [[-Identity] <Object>] [-Tenant <Object>] [-InMemory] [-SignInMode <Object>]
+ [-WhatIf] [-Confirm] [-Force] [-SearchOnCommonAreaPhoneMode <Object>] [-AsJob]

```

Comment:
I think you will add the type of parameters in the next release. If you need it we can help you.